.azure/pipelines/pr-validation-temp.yml

```diff
@@ -1,38 +0,0 @@
-# This configuration is temporary while we work on getting all unit tests to pass on PR checks
-
-# Don't run CI for this config
-trigger: none
-
-# Run PR validation on all branches
-pr:
-  branches:
-    include:
-    - '*'
-
-jobs:
-- template: jobs/default-build.yml
-  parameters:
-    jobName: Windows_Build
-    jobDisplayName: "Build only : Windows"
-    agentOs: Windows
-    buildArgs: '/p:SkipTests=true'
```

Comment:
Note this was ignored anyhow. See https://dev.azure.com/dnceng/public/_build/results?buildId=969282 and the warnings about `$(BuildScriptArgs)`. Might have a bug in this branch's version of default-build.yml

Reply:
Found it. I'll put up a quick PR later today to fix it.

Reply:
#29737